UploadToDB/get_food_prefs.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import json
+import pandas as pd
+from collections import Counter
+
+HEADER = {
+    "accept": "application/json",
+    "Content-Type": "application/json",
+}
+
+BASE_URL = "https://www.nexpo.arkadtlth.se/api"
+
+ADMIN_USER = {
+    "email": "admin@admin.com",
+    "password": "admin"
+}
+
+def _get_token():
+    url = f"{BASE_URL}/session/signin"
+    response = requests.post(url, headers=HEADER, data=json.dumps(ADMIN_USER))
+    return f"Bearer {response.json()['token']}"
```

Comment:
In the UploadToDB folder exists already a script named `login.py` that manages the login and returns the token. 
For consistency should we reuse it for all python scripts.